keras_core/ops/nn.py

```diff
@@ -993,6 +1026,38 @@ def compute_output_spec(self, target, output):
     ]
 )
 def binary_crossentropy(target, output, from_logits=False):
+    """Computes the binary cross-entropy loss between target tensor and output tensor.
+
+    The binary cross-entropy loss is commonly used in binary classification tasks
+    where each input sample belongs to one of the two classes. It measures the
+    dissimilarity between the target and output probabilities or logits.
+
+    Args:
+        target: The target tensor representing the true binary labels.
+            Its shape should match the shape of the `output` tensor.
+        output: The output tensor representing the predicted probabilities
+            or logits. Its shape should match the shape of the 
+            `target` tensor.
+        from_logits: (optional) Whether `output` is a tensor of logits or
+            probabilities.
+            Set it to `True` if `output` represents logits; otherwise,
+            set it to `False` if `output` represents probabilities. 
+            Default is `False`.
+
+    Returns:
+        Integer tensor: The computed binary cross-entropy loss between 
+        `target` and `output`.
+
+    Example:
+    ```
+    >>> target = keras_core.ops.convert_to_tensor([0, 1, 1, 0], 
+                                                  dtype=tf.float32)
```

Comment:
No reference to TF -- use `float32`

Note that you don't actually need to pass anything, the default is float32.